docs/Quickstart.md

```diff
@@ -1,3 +1,26 @@
+## Table of Contents
```

Comment:
Thanks, love having a TOC now.

Reply:
I figured for a long documentation it would be a convenient feature to be added. Glad it was of use!